walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +270,104 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeWithWholedayEvent() {
```

Comment:
For your tests, it'd be nice to add comments similar to the other tests to describe what you're testing for.